### PR TITLE
Upgrading buildtools ver to 1.0.25-prerelease-00025

### PR DESCRIPTION
**Pull request by venkat-raman251**
Upgraded the buildtools version to the version 1.0.25-prerelease-00023.
One more change is pending post this to get PerfTools to run. This is the bulk of the changes though.
/cc @mattellis @kcwalina @joshfree @jhendrixMSFT

### CODE REVIEW

File: src/System.Text.Formatting/src/System/Text/Formatting/FormattingData.cs [hunk 1]
```diff
@@ -131,6 +133,7 @@ private bool TryWriteDigitOrSymbol(ulong digitOrSymbolIndex, Span<byte> buffer,
             return true;
         }
 
+        [CLSCompliant(false)]
         public enum Symbol : ushort
```
Comment (KrzysztofCwalina) [inline src/System.Text.Formatting/src/System/Text/Formatting/FormattingData.cs]: I removed this enum earlier today.

**Comment by KrzysztofCwalina** [comment]
Besides the enum Symbol being added (after my removal earlier today), this looks good. Please feel free to merge after you clean up the enum issue.

**Comment by joshfree** [comment]
looks good. :+1:

**Comment by venkat-raman251** [comment]
Merged with changes that you made in the last few days. I got it to generate a report with the latest version of buildtools.
/cc @mattellis @kcwalina @joshfree

**Comment by KrzysztofCwalina** [comment]
Looks good

**Pull request merged**